src/test/globalSetup.test.ts

```diff
@@ -44,6 +45,9 @@ before(async () => {
     }
     const service = new DefaultTelemetryService(mockContext, mockAws, mockPublisher)
     ext.telemetry = service
+
+    // Warm up vscode before tests run
+    await vscode.extensions.getExtension(VSCODE_EXTENSION_ID.awstoolkit)!.activate()
```

Comment:
does this avoid a race?

Reply:
Yep, the first test that uses the extension activates it. That test often times out since activation takes a while.

Reply:
Was this activation moved from somewhere else? This is interesting because not all tests require the extension to be activated

Reply:
I'd been consistently getting a single test timeout when running from the IDE. I believe that was because that specific test was the first to activate the extension and we had a timeout of a few seconds.